__tests__/backend/taxjarApi.test.ts

```diff
@@ -0,0 +1,154 @@
+import _default from "taxjar/dist/util/request";
```

Comment:
I don't think we should import stuff from `dist` - it can fail unexpectedly if taxjar changes something in their bundler config
```suggestion
import _default from "taxjar/lib/util/request";
```